docs/sections/user_guide/yaml/components/mpas.rst

```diff
@@ -0,0 +1,98 @@
+.. _mpas_yaml:
+
+mpas
+====
+
+Structured YAML to run the WRF preprocessing component ``mpas`` is validated by JSON Schema and requires the ``mpas:`` block, described below. If ``mpas`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML ``mpas:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/mpas.yaml
+
+When MPAS is built from source code, static data input files and default namelist and streams files are also created. Here we show an example of how an arbitrarily named ``user:`` section may be used to define helpful variables that are heavily reused through the rest of the configuration. In this example, the ``mpas_app:`` entry is the path to where the MPAS model was installed.
+
+
+
+UW YAML for the ``mpas:`` Block
+-------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+
+boundary_conditions:
+^^^^^^^^^^^^^^^^^^^^
+
+Describes the boundary condition files needed for the forecast. These will be the output from the
+``init_atmosphere`` executable, which may be run using the ``mpas_init`` UW driver. Please see its
+documentation :ref:`here <mpas_init_yaml>`.
+
+interval_hours:
+"""""""""""""""
+
+Frequency interval of the given files, in integer hours.
+
+offset:
+"""""""
+
+How many hours earlier the external model used for boundary conditions started compared to the desired forecast cycle, in integer hours.
+
+path:
+"""""
+
+An absolute path to the MPAS-ready files to be used for initial and lateral boundary conditions input. The names of the files are specified in the MPAS ``streams.atmosphere`` XML file, and may be specified in the ``streams: values:`` block of the driver YAML.
+
+length:
+^^^^^^^
+
+The length of the forecast in integer hours.
+
+files_to_copy:
+^^^^^^^^^^^^^^
+
+See :ref:`this page <files_yaml>` for details.
+
+To reduce duplication of information in this section, it may be helpful to template the file that depends on the number of cores being used to run the executable. For example, instead of:
+
+.. code-block:: text
+
+   mpas:
+     files_to_copy:
+       conus.graph.info.part.32: /path/to/conus.graph.info.part.32
+
+Jinja expressions can be used to reference the number of cores used in execution:
```

Comment:
```suggestion
Jinja2 expressions can be used to reference the number of cores used in execution:
```
Annoying, I now. The Jinja2 people have a GitHub Issue where they discussed trying to change the name officially from Jinja2 to just Jinja and decided not to.